Gemfile

```diff
@@ -112,6 +112,7 @@ group :test do
   gem 'factory_bot'
   gem 'govuk-content-schema-test-helpers'
   gem 'maxitest'
+  gem 'minitest', '5.10.3'
```

Comment:
A comment here would be ideal, as this can probably be removed once Rails has been updated. It will also help stop people from letting dependabot upgrade the version!